### ISSUE TITLE
TCP JSON parser does not work for 64bit integers

### ISSUE THREAD

**Issue by jt185133** [fluent/fluent-bit#3481]
## Bug Report

**Description**
When the TCP input plugin receives JSON data any integer values larger than the 32bit max will be treated as 32bit max.

**Steps to Reproduce**

1. run fluent-bit with the following configuration:
```
[INPUT]
    Name        tcp
    Listen      0.0.0.0
    Port        5170
    Format      json

[OUTPUT]
    Name        stdout
    Match       *
```
2. Sending `'{"key":2147483000}'` via TCP yields expected output `[0] tcp.0: [1620920399.621495000, {"key"=>2147483000}]`
3. Sending `'{"key":2147483999}'` via TCP yields incorrect output `[0] tcp.0: [1620920522.688896000, {"key"=>2147483647}]`

**Expected Behavior**

Sending `'{"key":2147483999}'` via TCP should yield `[0] tcp.0: [1620920522.688896000, {"key"=>2147483999}]`

**Environment**

fluent-bit v1.7.4 running on Win10 v1909

**Comment by jt185133** [comment]
Note that it behaves correctly when running on Linux (CentOS 7), so it appears to be a problem specific to the Windows build.

**Comment by fujimotos** [comment]
> Note that it behaves correctly when running on Linux (CentOS 7), so it appears to be a problem specific to the Windows build

@jt185133  I have just posted a fix of this bug to #3501. 

With this applied, I can confirm that 64-bit numbers properly:

```powershell
PS> \bin\fluent-bit.exe -i tcp -p"format=json" -o stdout
...
[0] tcp.0: [1621327585.776476500, {"a"=>2147483999}]
```
